Components/badge/badge.css

```diff
@@ -0,0 +1,77 @@
+@import url(../avatar/avatar.css);
+:root {
```

Comment:
Use `:root` only in main css file ( component.css)

Reply:
Ok, will change that 👍